packages/vuetify/src/components/VBadge/__tests__/VBadge.spec.tsx

```diff
@@ -0,0 +1,54 @@
+// Components
+import { VBadge } from '..'
+
+// Utilities
+import { mount } from '@vue/test-utils'
+import { createVuetify } from '@/framework'
+
+describe('VBadge', () => {
+  const vuetify = createVuetify()
+
+  function mountFunction (options?: any) {
+    return mount(VBadge, {
+      ...options,
+      global: { plugins: [vuetify] },
+    })
+  }
+
+  it.each([
+    [{ modelValue: true }],
+    [{ modelValue: false }],
+    [{ floating: true }],
+    [{ dot: true }], // takes priority over badge slot
+    [{ dot: true, floating: true }],
+    [{ floating: true }],
+    [{ icon: 'foo' }], // badge slot overrides icon
+    [{ inline: true }],
+    [{ location: 'bottom-left' }],
+    [{ location: undefined }],
+    [{ location: null }],
+    [{ content: '10', max: 9 }],
+    [{ content: '10', max: 11 }],
+    [{ content: 'foo', max: 9 }],
+  ])('should render component and match snapshot with badge slot', (props: any) => {
```

Comment:
if possible please improve this title so that the snapshots are easier to distinguish